.github/workflows/reusable-test.yml

```diff
@@ -259,7 +279,7 @@ jobs:
         files: c:/dumps/${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}/*.dmp
 
     - name: Upload any crash dumps
-      if: (failure()) && (inputs.gather_dumps == true)
+      if: (steps.check_dumps.outputs.files_exists == 'true') && (inputs.gather_dumps == true)
```

Comment:
Please take a second look.

Reply:
Needs to be "always()" instead of "failure" to catch the case where the test reports success, but a crash dump is generated.